package.json

```diff
@@ -4,16 +4,21 @@
   "description": "Administrator dashboard for the agriculture software stack",
   "author": "agriculture-capstone",
   "private": true,
+  "license": "UNLICENSED",
   "scripts": {
     "dev": "node build/dev-server.js",
     "start": "node build/dev-server.js",
     "build": "node build/build.js",
-    "unit": "cross-env BABEL_ENV=test karma start test/unit/karma.conf.js --single-run",
-    "test": "npm run unit"
+    "test": "npm run test:unit:single && npm run test:e2e",
+    "test:unit": "cross-env BABEL_ENV=test karma start test/unit/karma.conf.js",
+    "test:unit:single": "npm run test:unit -- --single-run",
+    "test:e2e": "npm run test:e2e:single",
+    "test:e2e:single": "echo 'No e2e tests'"
```

Comment:
Hmm why is it echo-ing a message? Do we have to remember to change this?

Reply:
We should change this when we actually have e2e tests to run